src/components/Navigation.vue

```diff
@@ -14,5 +29,157 @@ import Vue from 'vue';
 export default Vue.extend({
   name: 'Navigation',
   components: {},
+  data: {
+    show: true,
+  },
+  methods: {
+      pickActive(element: any) {
+    const divs = ['#about', '#recap', '#faq', '#contact', '#proj', '#aboutDrop', '#recapDrop', '#faqDrop', '#contactDrop', '#projDrop']; // tslint:disable-line
+    for (let i = 0; i < divs.length; i++) { // tslint:disable-line
+        const header = document.querySelector(divs[i]); // tslint:disable-line
+        if (header != null) {header.classList.remove('active'); }
+    }
+    let header = document.querySelector(element);
+    if (header != null) { header.classList.add('active'); }
+    header = document.querySelector(element + 'Drop');
+    if (header != null) { header.classList.add('active'); }
+},
+handleSCroll(event: any) {
```

Comment:
can you add the specific type of the event param

Reply:
no, takes multiple events/types so doesn't work

Reply:
EventListener doesnt work?

Reply:
no it doesnt work with created or destroyed and I need those for the the function to work right

Reply:
alright that's fair, otherwise LGTM